services/web/server/src/simcore_service_webserver/wallets/_events.py

```diff
@@ -43,6 +44,17 @@ async def _auto_add_default_wallet(
                 created_at=wallet.created,
             )
 
+        preference_id = (
+            preferences_api.PreferredWalletIdFrontendUserPreference().preference_identifier
```

Comment:
I find very strange way to create an id :-) but I guess that is how it was designed by @GitHK

Reply:
Yes, I just reused what is there :) I think @GitHK/we didn't though of this use-case at that time.